### ISSUE TITLE
configure error: dracut version

### ISSUE THREAD

**Issue by giufus** [dracut-crypt-ssh/dracut-crypt-ssh#30]
Hi, 

I have the following error running configure script:

```
$ ./configure 
Searching for dracut... /usr/bin/dracut
Checking dracut version... 048
Checking dracut module prefix... /usr/lib/dracut/modules.d
Checking whether crypt-settle patch is needed... no
Checking for old-style modules... no
Checking for root's home... /root
Checking for libnss_files.so ... /lib64/libnss_files.so.2
./configure: line 117: [[: 048: value too great for base (error token is "048")
```

I'm trying to install from source using Fedora 28.

**Comment by ksyz** [comment]
Leading zeros from version number have to be dropped, so bash won't cofuse them for base8 numbers.

```
-       DRACUT_VERSION=$(echo ${DRACUT_VERSION} | cut -f 1 -d " ")
+       DRACUT_VERSION=$(echo ${DRACUT_VERSION} | cut -f 1 -d " " | sed -e 's/^00*//g' )
```
or
```
-DRACUT_VERSION=$(${DRACUT} --help | egrep "Version:[ \t]+[0-9]+" | egrep -o [0-9]+)
+DRACUT_VERSION=$(${DRACUT} --help | egrep "Version:[ \t]+[0-9]+" | egrep -o [1-9][0-9]+)
```

**Comment by bearcodi** [comment]
Worked for me on Fedora 29, thanks @ksyz

**Comment by strasharo** [comment]
Hitting the same issue on current Fedora 29, the solution proposed by @ksyz resolved it for me.

**Comment by mhavrlent** [comment]
Same issue on Fedora 31 Server. Thank you for the fix @ksyz.

**Comment by ahesford** [comment]
Fixed in 1b1f3fa90ddd
